src/transformers/feature_extraction_sequence_utils.py

```diff
@@ -187,23 +187,6 @@ def pad(
         padding_strategy = self._get_padding_strategies(padding=padding, max_length=max_length)
 
         required_input = processed_features[self.model_input_names[0]]
-        if required_input and not isinstance(required_input[0], np.ndarray):
```

Comment:
Leftover logic from the old implementation, `required_input[0]` is always `np.ndarray` now

Reply:
good catch!